auto_updater.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+git ls-remote --exit-code fork > /dev/null 2>&1
+if test $? = 1; then
+  git remote add fork git@github.com:adoptopenjdk-github-bot/homebrew-openjdk.git
+fi
+
+command -v hub > /dev/null 2>&1
+if test $? = 1; then
+  echo "Installing hub"
+  os=$(uname | awk '{print tolower($0)}')
+  wget --quiet "https://github.com/github/hub/releases/download/v2.12.3/hub-$os-amd64-2.12.3.tgz"
```

Comment:
Can we get the latest as opposed to a specific version?

Reply:
unfortunately not, we can install hub on the machine and then this function gets skipped anyway.